orttraining/orttraining/python/training/ortmodule/_training_manager.py

```diff
@@ -174,27 +118,23 @@ def backward(ctx, *grad_outputs):
                 for input_name in self._graph_info.user_input_names:
                     # Append to the results the backward output for each input that required grad
                     if input_name in require_grad_names_set:
-                        results.append(_utils._torch_tensor_from_dl_pack(
-                            backward_outputs.dlpack_at(require_grad_names_index),
-                            backward_outputs[require_grad_names_index]))
+                        results.append(backward_outputs[require_grad_names_index])
```

Comment:
what is the reasoning for this change? AFAIK this code was meant to keep high performance